README.md

```diff
@@ -1,3 +1,49 @@
-# tsuru-performance-tests
+# Tsuru Performance test suite
+
+The Tsuru test suite is a set of jmeter-ruby-dsl scripts describing a set of performance testing scenarios.
+They each can be found in the `dsl` folder and run locally or exported to jmx files for running on Blazemeter,
+Flood.io or on your own servers, to provide more resources to a performance test.
+
+Currently, these tests are only runnable against the CI environment.
+
+## How to install
+
+```
+Brew install jmeter
+git clone https://github.com/alphagov/tsuru-performance-testing
+cd ./tsuru-performance-testing
+bundle install
+```
+
+## How to run these tests locally
+
+To run the tests:
+
+```
+ruby deploy_apps.rb ruby deploy_apps.rb <TSURU_TARGET_URL> 100 100
```

Comment:
This command should be prefixed with `bundle exec` to use the dependencies installed with `bundle install` in the commands above.